### PR TITLE
mubert-base.commons-lang.mid-482.idx-78971.13.mutant

### PR DESCRIPTION
**Pull request by Acesif**

### CODE REVIEW

**Review by mruchann — COMMENTED** [review]
I was going to open a pull request about this, but then it would become duplicate PRs. I think your suggestion is wrong, you shouldn't add another ampersand.

File: commons-lang/src/main/java/org/apache/commons/lang3/LocaleUtils.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ private static boolean isISO3166CountryCode(final String str) {
      * @return true, if the given String is a ISO 639 compliant language code.
      */
         private static boolean isISO639LanguageCode(final String str) {
-            return StringUtils.isAllLowerCase(str) &(str.length() == 2 || str.length() == 3);
+            return StringUtils.isAllLowerCase(str) && (str.length() == 2 || str.length() == 3);
```
Comment (mruchann) [inline commons-lang/src/main/java/org/apache/commons/lang3/LocaleUtils.java:140]: `return StringUtils.isAllLowerCase(str) & (str.length() == 2 || str.length() == 3);`

Comment (Acesif) [reply commons-lang/src/main/java/org/apache/commons/lang3/LocaleUtils.java:140]: Thank you @mruchann for your suggestion, can you please explain why my suggestion is wrong? I think a singular ampersand is a bitwise AND and a pair of ampersand is the logical AND.
I also looked into the other methods in the source file whose return type was boolean, and those methods used `&&`. From that context I put double ampersand

Comment (Acesif) [reply commons-lang/src/main/java/org/apache/commons/lang3/LocaleUtils.java:140]: Thanks for pointing it out @mruchann

**Comment by darko-marinov** [comment]
@Acesif @mruchann how does this diff from `main`? What's the mutation?

**Comment by mruchann** [comment]
Here's the main branch:

```
private static boolean isISO639LanguageCode(final String str) {
        return StringUtils.isAllLowerCase(str) && (str.length() == 2 || str.length() == 3);
}
```

Screenshot 2024-06-23 at 4 44 55 AM

You're right. The task was only to format. So we should put only one ampersand again, and add extra space. Thank you 🚀

**Comment by mruchann** [comment]
As there's no mutation, this PR becomes wrong, right? It should be like this:

```
private static boolean isISO639LanguageCode(final String str) {
        return StringUtils.isAllLowerCase(str) & (str.length() == 2 || str.length() == 3);
}
```

**Comment by darko-marinov** [comment]
Does this even compile?

**Comment by mruchann** [comment]
Screenshot 2024-06-23 at 5 17 11 AM

It does.

**Pull request merged**